core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -473,12 +473,55 @@ class GroupCoordinator(val brokerId: Int,
     }
   }
 
+  private def onGroupUnloaded(groupEmigrating: Boolean)(group: GroupMetadata) {
```

Comment:
Could we move these two functions into GroupMetadataManager to avoid function passing with separate arguments?